src/components/Books/Form/Form.jsx

```diff
@@ -0,0 +1,42 @@
+import './Form.css'
+import Buttons from "../../Buttons/Buttons";
+
+const Form = () => {
+  return (
+    <div className='form-container'>
+      <h2>ADD NEW BOOK</h2>
+      <form
+      class="form flex"
+      method="post">
+        <div class="input flex">
+          <label for="book name">
+            <input
+              name="fullname"
+              class="wf"
+              type="text"
+              placeholder="Book Title"
+              maxlength="30"
+              id="book-name"
+              required
+            />
```

Comment:
- [OPTIONAL] The value of the label's attribute `for` should match the value of the input's `id` to make a correct relation between them.